src/generators/integer/index.ts

```diff
@@ -1,18 +1,32 @@
 import {Chance} from 'chance';
 
+/**
+ * By default, Chance.js would use range -9007199254740991 (-2^53 - 1) to 9007199254740991 (2^53 - 1).
+ * See https://chancejs.com/basics/integer.html.
+ *
+ * However, per the GraphQL spec, Integers are only treated as valid when they are representable as
+ * a 32-bit signed integer, providing the broadest support across platforms.
```

Comment:
My only suggestion would be adding the spec link to this comment.

https://spec.graphql.org/June2018/#sec-Int